### ISSUE TITLE
JOSM Statistics

### ISSUE THREAD

**Issue by jjiglesiasg** [piebro/OpenStreetMap-Statistics#42]
Considering that JOSM is the only one editor that allow external data import to OSM. I would like to see how many of the theoretical data produced by JOSM is actual data Made/Mapped by the contributors or data Imported by the contributors. It could not be realistic that several contributors has less than 1 thousand historical changesets and several millions of map changes at the same time.

**Comment by piebro** [comment]
Yes, it might be interesting to distinguish between manual contributions and import contributions with external data. I'm not sure how to do it though. There might be tag in JOSM to indicate (maybe possible to detect with #49) or maybe a heuristic if the number of edits per changeset is to high, but I'm not really sure if this would make sense.

**Comment by piebro** [comment]
Looking at tags of JOSM (https://piebro.github.io/openstreetmap-statistics/#2fb7) I don't see an easy way to distinguish between data imports and and manual added data with JOSM. If anyone has a good idea feel free to add it here.

**Comment by westnordost** [comment]
Maybe look for certain keywords in the changeset comment (such as "import", but maybe not "cleanup import"). It is a fuzzy match, but single false positives and false negatives will hopefully even out in the statistics?

(Edit: plus a large extent of the bbox. Usually, I'd think that imports are generally over larger areas, i.e. at least village/municipality level)

**Comment by piebro** [comment]
That's some helpful info. I haven't looked at the comments at all until now, but this might be a good start together with. Looking at the bbox also sound like a good idea.
